src/lib/client/store/activities.ts

```diff
@@ -71,4 +71,8 @@ export function isActive(a: Activity) {
 
 export function isComplete(a: Activity) {
   return !!a.endTime;
+}
+
+export function getActivityPath(activity: Activity) {

```

Comment:
I feel like getActivityPath should return the full  path, including the ID. Simplifies the calling code more and still feels natural.